risedev.yml

```diff
@@ -280,6 +280,7 @@ risedev:
     - use: meta-node
       unsafe-disable-recovery: true
     - use: compute-node
+      tiered-cache-uri: "file:///risingwave/.risingwave/data/filecache"
```

Comment:
Would be better to use relative path, because developers will start this profile on their local machine.

Reply:
Also I think a better way to configure this is to have a config like

```
enable-tiered-cache: true
```

in RiseDev. When generating config, RiseDev will automatically create directory in `compute_node_service.rs`.

Reply:
> Also I think a better way to configure this is to have a config like
> 
> ```
> enable-tiered-cache: true
> ```
> 
> in RiseDev. When generating config, RiseDev will automatically create directory in `compute_node_service.rs`.

I don't fully understand you. 🤔 Then how to pass the generated directory to each compute node?

Reply:
As cli arg